exercise_2/WeightMatrix.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+#include <iostream>
+#include <vector>
+
+struct WeightMatrix
+{
+public:
+	WeightMatrix();
+	WeightMatrix(const int& vertices);
+	~WeightMatrix();
+	std::vector<std::vector<int>> weightMatrix;
```

Comment:
I would rather suggest to keep struct variables in private section and access to it via getters and setters